### PR TITLE
document connection between tokens and docs

### PR DESCRIPTION
**Pull request by wiese**
Bug: [T256024](https://phabricator.wikimedia.org/T256024)

### CODE REVIEW

File: tokens/README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+## Package organization
+
+We organized the tokens in three categories
+* globals
+* aliases
+* component-level
+
+Each category serves its own purpose.
+
+    ⚠️ TBD
```
Comment (wiese) [inline tokens/README.md:14]: I guess @SaiSan-WMDE may want to chime in here in a follow-up commit.

Comment (SaiSan-WMDE) [reply tokens/README.md:14]: ""Global tokens reflect the elementary visual foundations of the system. They are context agnostic (e.g. "accent-30"). Aliases are abstractions that communicate the intended context of use of a token. They mostly represent styles that are shared among several components (e.g. "border-color-progressive-active"). Finally, component-level tokens are used to represent every value associated with the style of a component (e.g."button-primary-progressive-border-color-active").""

Let me know if this matches expectations, and if it's not too wordy.

Comment (wiese) [reply tokens/README.md:14]: Makes sense to me and matches what we discussed so far.

Given you have the tools and skills, feel free to turn this into an edit on this branch.
You can do this e.g. also (we live in the future) via the "insert a suggestion" feature right here in the comment editor.
